File: backend/ws_wa_store_ccadmin/src/main/java/com/ccadmin/app/sale/model/entity/PresaleChannelEntity.java
package com.ccadmin.app.sale.model.entity;

import com.ccadmin.app.shared.model.entity.AuditTableEntity;
import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;

import java.io.Serializable;

@Entity
@Table(name = "presale_channel")
public class PresaleChannelEntity extends AuditTableEntity implements Serializable {

    @Id
    public String PresaleCod;
    public String ChannelCod;

    public PresaleChannelEntity() {
    }
}

File: frontend/web_ccadmin_lt/src/app/enterprise/sale/model/entity/PresaleChannelEntity.ts
import { AuditTableEntity } from "src/app/enterprise/shared/model/entity/AuditTableEntity";
import { SaleConstants } from "../constants/SaleConstants";

export class PresaleChannelEntity extends AuditTableEntity
{
    public PresaleCod: string = "";
    public ChannelCod: string = SaleConstants.COMMERCIAL_CHANNEL_IN_PERSON;

    public constructor()
    {
        super();
    }
}
